test/integrations/destinations/tune/processor/trackTestData.ts

```diff
@@ -0,0 +1,225 @@
+import { Destination } from '../../../../../src/types';
+import { ProcessorTestData } from '../../../testTypes';
+import {
+  generateMetadata,
+  generateSimplifiedTrackPayload,
+  overrideDestination,
+  transformResultBuilder,
+} from '../../../testUtils';
+
+const destination: Destination = {
+  ID: '123',
+  Name: 'tune',
+  DestinationDefinition: {
+    ID: '123',
+    Name: 'tune',
+    DisplayName: 'tune',
+    Config: {},
+  },
+  Config: {
+    connectionMode: {
+      web: 'cloud',
+    },
+    consentManagement: {},
+    oneTrustCookieCategories: {},
+    ketchConsentPurposes: {},
+    tuneEvents: [
+      {
+        url: 'https://demo.go2cloud.org/aff_l?offer_id=45&aff_id=1029',
+        eventName: 'Product added',
+        standardMapping: [
+          { to: 'aff_id', from: 'affId' },
+          { to: 'promo_code', from: 'promoCode' },
+          { to: 'security_token', from: 'securityToken' },
+          { to: 'status', from: 'status' },
+          { to: 'transaction_id', from: 'mytransactionId' },
+        ],
+        advSubIdMapping: [{ from: 'context.ip', to: 'adv_sub2' }],
+        advUniqueIdMapping: [],
```

Comment:
`advUniqueIdMapping` is not getting covered can you please add some mapping here as well as in payload